cfme/tests/infrastructure/enable_c_u_roles.py

```diff
@@ -0,0 +1,7 @@
+from cfme.configure.configuration import server_roles_enabled
```

Comment:
This isn't used anywhere is it? C&U enablement is already provided I think elsewhere.